app/javascript/oldjs/controllers/ansible_repository/repository_form_controller.js

```diff
@@ -77,6 +77,7 @@ ManageIQ.angular.app.controller('repositoryFormController', ['repositoryId', 'mi
 
   var getRepositoryFormData = function(response) {
     var data = response;
+    data.verify_ssl = data.verify_ssl == 1 ? true : false;
```

Comment:
can you do:

```suggestion
    data.verify_ssl = data.verify_ssl == 1;
```

Reply:
I'm fine merging without this change - it's up to you

Reply:
Nah, it is simpler way to do the same this so I can make the change.  Good call/suggestion.